README.md

```diff
@@ -1,10 +1,30 @@
-# mumble-pahelper
+<p align="center"><img src = "resources/MumblePAHelper.png"/></p>
 
 Mumble Positional Audio Helper is a small tool for testing the functionality
 of positional audio plugins outside of Mumble.
 
 You can use this tool to quickly see if your plugin is working as it should.
-Simply place it next to your plugin (or one level up from a /plugins folder
-and launch it.
 
-![Screenshot of pahelper](https://github.com/mumble-voip/mumble-pahelper/raw/master/media/MumblePAHelper.png)
\ No newline at end of file
+Simply place it next to your plugin and launch it.
```

Comment:
I don't know if it's better to remove it, put the sentence in the same paragraph or use yours...